### PR TITLE
Basic functionality to explore page and homepage

### PR DESCRIPTION
**Pull request by deekshithmd**
Added functionality to homepage and explore page, please review it and provide your comments

Main files to check:
Home
Explore 
Single Video 

other files
Liked
History
WatchLater

### CODE REVIEW

File: src/components/Video/Video.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      {/* <iframe
+        width="100%"
+        height="100%"
+        src={`https://www.youtube.com/embed/${id}`}
+        title="YouTube video player"
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture;"
+      ></iframe> */}
+    </>
+  );
```
Comment (mayuresh-tech) [inline src/components/Video/Video.jsx:21]: You can remove commented code if not required

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

File: src/components/SingleVideo/SingleVideo.jsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                    <div className="video-details text-md text-bold">
+                      <div className="video-header">
+                        <span className=" video-title text-justify">
+                          {video.title}
+                        </span>
+                        <i className="fa-solid fa-ellipsis-vertical options"></i>
+                      </div>
+                      <div className="video-footer text-sm">
+                        <span>{video.creator}</span>
+                        <span>12/05/2018</span>
```
Comment (saurabhchirde) [inline src/components/SingleVideo/SingleVideo.jsx]: I think you have written this hardcoded date by mistake.

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issue, your code is clean and well written. You can proceed to merge.

File: src/components/SingleVideo/SingleVideo.jsx (hunk 2 above)
Comment (deekshithmd) [reply src/components/SingleVideo/SingleVideo.jsx]: yes, thank you for pointing it out...

**Pull request merged**